### PR TITLE
Add missing dbghelp.dll to extensions folder

### PR DESCRIPTION
**Pull request by smadala**
Fixes: https://github.com/Microsoft/vstest/issues/916

### CODE REVIEW

File: test/TestAssets/CPPSimpleTestProject/CPPSimpleProj/CPPSimpleProj.vcxproj [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <ProjectGuid>{076CE7E6-92E8-49FD-9D98-57D377FAA46E}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>CPPSimpleProj</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.10586.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
```
Comment (Faizan2304) [inline test/TestAssets/CPPSimpleTestProject/CPPSimpleProj/CPPSimpleProj.vcxproj]: Why did we need to update this file?

File: test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj [hunk 2]
```diff
@@ -22,7 +22,7 @@
       <Version>$(ChutzpahAdapterVersion)</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.TestPlatform.TestAsset.NativeCPP">
-      <Version>1.0.0</Version>
+      <Version>2.0.0</Version>
```
Comment (Faizan2304) [inline test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj]: Please make sure we are creating Nuget package of Microsoft.TestPlatform.TestAsset.NativeCPP with version 2.0.0

**Review by Faizan2304 — APPROVED** [review]
LGTM 👍

**Review by singhsarab — APPROVED** [review]

File: test/TestAssets/CPPSimpleTestProject/CPPSimpleProj/CPPSimpleProj.vcxproj (hunk 1 above)
Comment (smadala) [inline test/TestAssets/CPPSimpleTestProject/CPPSimpleProj/CPPSimpleProj.vcxproj]: Updated the required windows sdk to compile c++ project.

File: test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj (hunk 2 above)
Comment (smadala) [inline test/Microsoft.TestPlatform.AcceptanceTests/Microsoft.TestPlatform.AcceptanceTests.csproj]: Added.

**Pull request merged**